governance/votes/Executive vote - July 25, 2024.md

```diff
@@ -0,0 +1,133 @@
+  - The [Emergency Shutdown Module (ESM)](https://docs.makerdao.com/smart-contract-modules/shutdown/the-emergency-shutdown-process-for-multi-collateral-dai-mcd) [Minimum Threshold](https://docs.makerdao.com/smart-contract-modules/shutdown/emergency-shutdown-module) will be increased.
+  - The LITE-PSM-USDC-A Keeper Network Job will be deployed.
+  - `jar` will be added to the Chainlog.
+  - A Spark Proxy Spell will be triggered.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### LITE-PSM-USDC-A Phase 1 Actions
+
+All the actions described below are part of the [first phase](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644/1) in the migration process [from PSM-USDC-A to LITE-PSM-USDC-A](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512).
+
+#### PSM-USDC-A DC-IAM Parameters Update
+
+- **Autorization:** [Poll 1124](https://vote.makerdao.com/polling/QmdcHXHy), [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644/2)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644/1)
+
+If this executive proposal passes, then the PSM-USDC-A DC-IAM parameter below will be updated as follows:
+
+- The [Target Available Debt (`gap`)](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-2-target-available-debt-gap-) will be decreased by 20 million from 400 million to **380 million**.
+
+#### LITE-PSM-USDC-A DC-IAM Setup
+
+- **Autorization:** [Poll 1124](https://vote.makerdao.com/polling/QmdcHXHy), [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644/2)
+- **Proposal**: [Forum post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644/1)
+
+If this executive proposal passes, then the LITE-PSM-USDC-A DC-IAM will be set up as follows:
+
+- [Maximum Debt Ceiling (`line`)](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512#lite-psm-usdc-a-parameters-overview-8): 50 million DAI
```

Comment:
Should we use the definitions in the [Atlas](https://mips.makerdao.com/mips/details/MIP104#14-3-1-4-1-maximum-debt-ceiling-line-) instead of the forum post?